pulp_python/app/models.py

```diff
@@ -33,6 +37,41 @@ class PythonDistribution(PublicationDistribution):
 
     TYPE = 'python'
 
+    def content_handler(self, path):
+        """
+        Handler to serve extra, non-Artifact content for this Distribution
+
+        Args:
+            path (str): The path being requested
+        Returns:
+            None if there is no content to be served at path. Otherwise a
+            aiohttp.web_response.Response with the content.
+        """
+        path = PurePath(path)
+        name = None
+        version = None
+        if path.match("pypi/*/*/json"):
+            version = path.parts[2]
+            name = path.parts[1]
+        elif path.match("pypi/*/json"):
+            name = path.parts[1]
+        if name:
+            try:
+                package_content = PythonPackageContent.objects.filter(
+                    pk__in=self.publication.repository_version.content,
+                    name__iexact=name
+                )
+                # TODO Change this value to the Repo's serial value when implemented
+                headers = {PYPI_LAST_SERIAL: str(PYPI_SERIAL_CONSTANT)}
+                return json_response(python_content_to_json(self.base_path,
+                                                            package_content,
+                                                            version=version),
+                                     headers=headers)
+            finally:
+                pass
```

Comment:
Is there a reason to keep this?

Reply:
Initially it use to be a bare exception catch, but the linter keep yelling so I switched it to finally.  I don't think it's necessary to wrap it in a try/except block, but there are two database queries so I am not sure if they have potential for errors.